src/sbt-test/skip-windows/caching/test

```diff
@@ -183,15 +183,6 @@ $ exec chmod 000 src/main/scala/Valid.scala
 -> scalafix SyntacticRule
 $ delete src/main/scala
 
-# files should be re-checked if a custom tool classpath is used (even if the rule is the same)
-> set scalafixConfig := None
-$ mkdir src/main/scala
-$ copy-file files/Valid.scala src/main/scala/Valid.scala
-> scalafix --tool-classpath=target/scala-2.12/classes RemoveUnused
-$ exec chmod 000 src/main/scala/Valid.scala
--> scalafix --tool-classpath=target/scala-2.12/classes RemoveUnused
-$ delete src/main/scala
-
```

Comment:
This was testing a limitation/workaround, that https://github.com/scalacenter/sbt-scalafix/pull/409 made obsolete. This PR exposed this as the first scalafix invocation now warms up the compilation cache and thus goes through.